app.rb

```diff
@@ -0,0 +1,172 @@
+require './book'
+require './teacher'
+require './student'
+require './rental'
+
+class App
+  attr_reader :books, :people, :rental
+
+  def initialize
+    @books = []
+    @people = []
+    @rental = []
+  end
+
+  def run
+    show_menu
+
+    menu_list = {
+      '1' => method(:book_list),
```

Comment:
- [ ] It will be great to return the `title` and the `author` when you are listing all the books. At the moment only the title is shown